### PR TITLE
#54515-2 - Fix checkbox styles after changes

### PR DESCRIPTION
**Pull request by jszczech**

### CODE REVIEW

File: src/molecules/checkbox/Checkbox.scss [hunk 1]
```diff
@@ -21,20 +21,18 @@ $checkbox__icon-transition   : $transition-base !default;
     position: absolute;
     opacity: 0;
 
-    &:checked + .a-checkbox__icon-wrapper .a-checkbox__icon {
+    &:checked + .a-checkbox__label .a-checkbox__icon-wrapper .a-checkbox__icon {
       opacity: 1;
     }
 
-    &:checked ~ .a-checkbox__icon-wrapper {
+    &:checked ~ .a-checkbox__label .a-checkbox__icon-wrapper {
```
Comment (szafran89) [inline src/molecules/checkbox/Checkbox.scss]: is it look better for you?
```
    &:checked + .a-checkbox__label {
      .a-checkbox__icon-wrapper {
        border: $checkbox__border--checked;
      }

      .a-checkbox__icon {
        opacity: 1;
      }
    }
```

File: src/molecules/checkbox/Checkbox.scss [hunk 2]
```diff
@@ -43,6 +41,7 @@ $checkbox__icon-transition   : $transition-base !default;
     align-items: center;
     width: $checkbox__item-size;
```
Comment (szafran89) [inline src/molecules/checkbox/Checkbox.scss]: sth wronk with variables names. Should be `$checkbox__icon-width` the same for other

File: src/molecules/checkbox/Checkbox.scss [hunk 3]
```diff
@@ -43,6 +41,7 @@ $checkbox__icon-transition   : $transition-base !default;
...
     height: $checkbox__item-size;
+    margin: $reset $spacer $reset $reset;
     border: $checkbox__border;
```
Comment (szafran89) [inline src/molecules/checkbox/Checkbox.scss]: `$checkbox__icon-wrapper-border`

**Comment by szafran89** [comment]
Instead of `margin-bottom: $checkbox__margin;` add `margin: $checkbox__margin;` and update checkbox marggin variable

**Comment by szafran89** [comment]
Please make sure if our test still look ok

**Comment by jszczech** [comment]
> Instead of `margin-bottom: $checkbox__margin;` add `margin: $checkbox__margin;` and update checkbox marggin variable

Actually... it was unnecessary, I removed it :D

File: src/molecules/checkbox/Checkbox.scss [hunk 4]
```diff
@@ -1,50 +1,50 @@
...
   &__icon-wrapper {
     display: flex;
     align-items: center;
-    width: $checkbox__item-size;
-    height: $checkbox__item-size;
-    border: $checkbox__border;
-    border-radius: $checkbox__icon-border-radius;
+    width: $checkbox__icon-wrapper-width;
+    height: $checkbox__icon-wrapper-height;
+    margin: $reset $spacer $reset $reset;
```
Comment (szafran89) [inline src/molecules/checkbox/Checkbox.scss]: please add variable `$checkbox__icon-wrapper-margin`

Comment (jszczech) [reply src/molecules/checkbox/Checkbox.scss]: done :)

**Pull request merged**